torchci/lib/bot/verifyDisableTestIssueBot.ts

```diff
@@ -10,17 +12,17 @@ export const disabledTestIssueTitle = new RegExp(
 );
 export const pytorchBotId = 54816060;
 
-export const supportedPlatforms = new Set([
-  "asan",
-  "linux",
-  "mac",
-  "macos",
-  "rocm",
-  "slow",
-  "win",
-  "windows",
-  "dynamo",
-  "inductor",
+export const supportedPlatforms = new Map([
+  ["asan", undefined],
```

Comment:
Overall LGTM, although I think it's nicer to have the value here as an array of label instead for flexibility, i.e.

```
export const supportedPlatforms = new Map([
  ["asan", []],
  ["linux", []],
  ["mac", ["module: macos"]],
  ["macos", ["module: macos"]],
  ["rocm", ["module: rocm"]],
  ["slow", []],
  ["win", ["module: windows"]],
  ["windows", ["module: windows"]],
  ["dynamo", ["oncall: pt2"]],
  ["inductor", ["oncall: pt2"]],
]);
```

The rest could be built around this.  Thoughts?